js/index.js

```diff
@@ -48,6 +48,7 @@ var Gamecircle = Class(function () {
 
 		var param = {"achievementID":achievementID,"percentSolved":percentSolved || 100};
 
+		achievementID = achievementID.replace('-', '');
```

Comment:
isn't it better to just send the correct achievement id from the application itself.